Telemetry radio - allowed frequencies (#2453)
PX4/PX4-user_guide: en/telemetry/README.md

You can however connect an LTE module to a companion computer and use it to route MAVLink traffic from the flight controller.
For more information see: [Companion Computer Peripherals > Data Telephony](../companion_computer/companion_computer_peripherals.md#data_telephony).
:::

## Allowed Frequency Bands

Radio bands allowed for use with drones differ between continents, regions, countries, and even states.
You should select a telemetry radio that uses a frequency range that is allowed in the areas where you plan on using the drone.

> Low power [SiK radios](../telemetry/sik_radio.md) such as the [Holybro Telemetry Radio](../telemetry/holybro_sik_radio.md) commonly provide 915 MHz and 433 MHz variants.

Author: ```suggestion
Low power [SiK radios](../telemetry/sik_radio.md), such as the [Holybro Telemetry Radio](../telemetry/holybro_sik_radio.md), are often available in 915 MHz and 433 MHz variants.
```
